### DISCUSSION TITLE
Extend auto-coincident feature to support snapping circle to surrounding points

### DISCUSSION THREAD

**Discussion by Intuos5** [hlorus/CAD_Sketcher#157] [Feature Requests]
### Description

When creating a circle, you cannot snap it to a point that is close to it to define it's radius.

https://user-images.githubusercontent.com/63149655/170656114-7e98e1e3-b9e1-4305-8c01-2ae78777977e.mp4

### Addon Version

0.22.6

### Blender Version

Blender 3.1

### What platform are you running on?

Windows

**Comment by hlorus** [comment]
IMO this is just a missing feature as the auto-coincident is currently implemented for points only. But i agree this would be a nice addition.
